src/System.Management.Automation/engine/remoting/commands/InvokeCommandCommand.cs

```diff
@@ -263,6 +263,33 @@ public override SwitchParameter UseSSL
             }
         }
 
+        /// <summary>
+        /// Gets or sets strict mode.
+        /// </summary>
+        [Experimental("PSStrictModeVersionAssignment", ExperimentAction.Show)]
+        [Parameter(ParameterSetName = InvokeCommandCommand.InProcParameterSet)]
+        public Version StrictModeVersion
+        {
+            get
+            {
+                return _strictmodeversion;
+            }
+
+            set
+            {
+                if (ExperimentalFeature.IsEnabled("PSStrictModeVersionAssignment"))
+                {
+                    _strictmodeversion = value;
+                }
+                else
+                {
+                    throw new ArgumentException("ExperimentalFeature 'PSStrictModeVersionAssignment' is not enabled");
+                }
```

Comment:
You can just do `_strictmodeversion = value;` here without the check. When this experimental feature is disabled, this property won't be shown up as a parameter.